nodewatcher/modules/devices/ubnt/unifi.py

```diff
@@ -0,0 +1,145 @@
+from django.utils.translation import ugettext_lazy as _
+
+from nodewatcher.core.generator.cgm import base as cgm_base, protocols as cgm_protocols, devices as cgm_devices
+
+
+class UBNTUapACLite(cgm_devices.DeviceBase):
+    """
+    UBNT UAP AC Lite device descriptor.
+    """
+
+    identifier = 'ub-uap-ac-lite'
+    name = "Unifi AP AC Lite"
+    manufacturer = "Ubiquiti"
+    url = 'http://www.ubnt.com/'
+    architecture = 'ar71xx'
+    radios = [
+        cgm_devices.IntegratedRadio('wifi0', _("Integrated wireless radio (5 GHz)"), [
+            cgm_protocols.IEEE80211AC(
+                cgm_protocols.IEEE80211AC.SHORT_GI_20,
+                cgm_protocols.IEEE80211AC.SHORT_GI_40,
+                cgm_protocols.IEEE80211AC.RX_STBC1,
+                cgm_protocols.IEEE80211AC.DSSS_CCK_40,
+            )
+        ], [
+            cgm_devices.AntennaConnector('a1', "Antenna1")
```

Comment:
Does it have only one antenna? Didn't you say that this is difference between models? How many antennas they have?